conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
+
 pdf.ruling.section.askForReview.title=Gofyn am adolygiad gan CThEF
-pdf.ruling.section.askForReview.email=E-bostiwch <strong>{0}</strong> yn gofyn am adolygiad a rhowch eich rhesymau cyn pen 30 diwrnod ar ôl cael yr e-bost am eich dyfarniad.
+pdf.ruling.section.askForReview.reviewOfficer=Bydd swyddog adolygu yn edrych ar y penderfyniad. Mae'r swyddog hwnnw yn ddiduedd. Bydd canlyniad yr adolygiad yn cael ei anfon atoch cyn pen 45 diwrnod.
+pdf.ruling.section.askForReview.ifYouDisagree=Os byddwch yn anghytuno â chanlyniad yr adolygiad, bydd yn dal i fod yn bosibl i chi apelio ar y tribiwnlys. Mae'n rhaid i chi wneud hyn cyn pen 30 diwrnod i ddyddiad y llythyr sy'n rhoi gwybod i chi am ganlyniad yr adolygiad. Ar ôl i adolygiad cael ei ddechrau, ni allwch apelio ar dribiwnlys hyd nes y bydd y swyddog adolygu'n rhoi gwybod i chi beth yw canlyniad ei adolygiad.
+pdf.ruling.section.askForReview.furtherInfo=I gael rhagor o wybodaeth am adolygiadau, darllenwch y dudalen ganlynol.
+pdf.ruling.section.askForReview.guidance.url=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy</a>
+pdf.ruling.section.askForReview.requestReview=I wneud cais am adolygiad, e-bostiwch, neu ysgrifennwch at, Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol CThEF (SOLS) yn uniongyrchol. Mae'r cyfeiriad e-bost a'r cyfeiriad post i'w gweld isod.
+pdf.ruling.section.askForReview.solsTeam=Gan fod y tîm SOLS yn rhan wahanol o CThEF, bydd angen yr wybodaeth ganlynol arnynt i gael hyd i'ch achos ac i ymgymryd â'r adolygiad:
+pdf.ruling.section.askForReview.solsTeam.1=eich enw, neu enw'r busnes, fel y dangosir ar frig y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.2=pob cyfeirnod a ddefnyddir ar frig y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.3=enw'r tîm neu'r swyddog i CThEF a enwir ar ddiwedd y llythyr hwn,
+pdf.ruling.section.askForReview.solsTeam.4=unrhyw wybodaeth newydd,
+pdf.ruling.section.askForReview.solsTeam.5=rhesymau pam rydych yn anghytuno â'r penderfyniad.
+pdf.ruling.section.askForReview.solsTeam.cannotWrite=Os oes yna reswm pam na allwch ysgrifennu at y tîm SOLS cyn pen 30 diwrnod i ddyddiad y llythyr hwn, bydd angen esgus rhesymol arnoch am yr oedi er mwyn cael estyniad. Bydd angen i chi ysgrifennu at y tîm SOLS cyn gynted â phosibl i esbonio'r oedi wrth dderbyn y cynnig o adolygiad. Os nad oes gennych esgus rhesymol, ni ellir ymgymryd â'r adolygiad. Gallwch ofyn i dribiwnlys dderbyn apêl hwyr.
+pdf.ruling.section.askForReview.solsTeam.email=reviews@hmrc.gov.uk
+pdf.ruling.section.askForReview.solsTeam.emailText=Cyfeiriad e-bost y tîm SOLS yw: <a class="govuk-link" href="mailto:{0}">{1}</a> Os byddwch yn penderfynu defnyddio e-bost i gyfathrebu, gweler yr wybodaeth bwysig ar ddiwedd y llythyr hwn. Os ydych am gysylltu â'r tîm yn ysgrifenedig, y cyfeiriad yw –
```

Comment:
Once`pdf.ruling.section.askForReview.solsTeam.email` has been removed, replace this `pdf.ruling.section.askForReview.solsTeam.emailText` with `pdf.ruling.section.askForReview.solsTeam.email`.

Also, hardcode the email `reviews@hmrc.gov.uk` in the message so it becomes: `Cyfeiriad e-bost y tîm SOLS yw: <a class="govuk-link" href="mailto:reviews@hmrc.gov.uk">reviews@hmrc.gov.uk</a> Os byddwch yn penderfynu defnyddio e-bost i gyfathrebu, gweler yr wybodaeth bwysig ar ddiwedd y llythyr hwn. Os ydych am gysylltu â’r tîm yn ysgrifenedig, y cyfeiriad yw –` thanks

Reply:
Please make sure to grab the above content which has been updated to show the apostrophe in `â’r`, thanks